data/Mitchell_2008/metadata.yml

```diff
@@ -0,0 +1,521 @@
+    Psi-leaf (Delta-psi-leaf ) were also included as a trait to explain shifts in
+    water availability during wet periods (early spring) and dry periods (early autumn).
+- var_in: DYleaf
+  unit_in: MPa
+  trait_name: .na.character
+  entity_type: population
+  value_type: mean
+  basis_of_value: measurement
+  replicates: 3-5
+  methods: Difference between summer and spring measurements; Seasonal differences
+    in pre-dawn leaf water potential were also included as a trait to explain shifts
+    in water availability during wet periods (early spring) and dry periods (early
+    autumn). Leaf water potential values were measured on the healthy, mature leaves
+    of 3-5 individuals of each species using two PMS-1000 pressure chambers (PMS Instrument
+    Corvallis, OR, USA). Pre-dawn measurements were made on clear, cloudless mornings
+    between 0300 and 0600 hours, depending on the season. Leaves or leafy twigs were
+    cut, bagged in plastic and transferred promptly to the pressure chamber for measurement.
+- var_in: Yleafmax
+  unit_in: MPa
+  trait_name: water_potential_predawn
+  entity_type: population
+  value_type: mean
+  basis_of_value: measurement
+  replicates: 3-5
+  methods: Spring-time (September) pre-dawn leaf water potential was assumed to be
+    representative of maximum or close to maximum values of leaf water status. Leaf
+    water potential values were measured on the healthy, mature leaves of 3-5 individuals
+    of each species using two PMS-1000 pressure chambers (PMS Instrument Corvallis,
+    OR, USA). Pre-dawn measurements were made on clear, cloudless mornings between
+    0300 and 0600 hours, depending on the season. Leaves or leafy twigs were cut,
+    bagged in plastic and transferred promptly to the pressure chamber for measurement.
+- var_in: gs min
+  unit_in: mmol{H2O}/m2/s
+  trait_name: .na.character
+  entity_type: population
+  value_type: mean
+  basis_of_value: measurement
+  replicates: '4'
+  temporal_context: summer
+  methods: Stomatal conductance (gs) measured in autumn (March) represent estimates
+    of minimum values of gs for this study. Stomatal conductance to water vapour (gs)
+    was measured at mid-morning between 1000 and 1200 hours as an estimate of daily
+    peak water loss, using the Licor 6400 gas exchange system (Licor Instruments,
+    Lincoln, NE, USA). Chamber CO2 concentration was set at 360 umol mol^-1 and photosynthetic
+    photon flux density (red-blue light source) at 1,500 umol m^-2 s^-1. Measurements
+    were made on two healthy, sun-lit and fully expanded leaves from each of four
+    individuals per species.
+- var_in: gs max
+  unit_in: mmol{H2O}/m2/s
+  trait_name: leaf_stomatal_conductance_per_area_at_Asat
+  entity_type: population
+  value_type: mean
+  basis_of_value: measurement
+  replicates: '4'
+  temporal_context: spring
+  methods: Stomatal conductance (gs) measured in autumn (March) represent estimates
```

Comment:
To me, this is the mean for that specific time of year - so it might be the individual max across the year, but it isn't the max once I have temporal context mapped in. And I've changed to `autumn`. Not sure why Gillian & I previously mapped that as summer.

Reply:
Sorry, value_type: mean was correct. I meant that in the manuscript it says gs max was measured in early spring, not autumn, to represent 'maximum' estimates for the study. I think the methods for gs max is repeated for gs min when they're slightly different.

Reply:
Fixed